runtime/j9vm/javanextvmi.cpp

```diff
@@ -351,7 +351,7 @@ JVM_VirtualThreadUnmountBegin(JNIEnv *env, jobject thread, jboolean lastUnmount)
 	f_monitorEnter(vm->liveVirtualThreadListMutex);
 	j9object_t threadObj = J9_JNI_UNWRAP_REFERENCE(thread);
 
-	while (J9OBJECT_I64_LOAD(currentThread, threadObj, vm->virtualThreadInspectorCountOffset) > 0) {
+	while (0 != J9OBJECT_I64_LOAD(currentThread, threadObj, vm->virtualThreadInspectorCountOffset)) {
 		/* Thread is being inspected, wait. */
```

Comment:
Is the below scenario possible? Can we mount while unmounting?
```suggestion
		/* Thread is being inspected or mounted, wait. */
```

Reply:
Leaving as is per offline discussion